.npmignore

```diff
@@ -0,0 +1,26 @@
+# https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package
+
+# build tool config
+Jenkinsfile
+.travis.yml
+
+# code examples
+examples
+
+# docs
+CONTRIBUTING.md
+
```

Comment:
IIRC the `README` is displayed on the npm site, but I don't think `CHANGES` is used there perhaps we need not include that in the package either?